.luacheckrc

```diff
@@ -1,5 +1,5 @@
 std = 'ngx_lua'
-globals = { 'ngx' }
+globals = { 'ngx', 'os.getenv' }
```

Comment:
tip: we can avoid having to do this is we load the `os` module locally, as done in #198

Reply:
good tip